src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Cat;
+import core.basesyntax.model.Dog;
+import java.time.LocalDate;
+
+public class Main {
```

Comment:
Well done for noticing the incorrect class name. But I think such renaming is quite radical.
I think renaming `Hello_World` -> `HelloWorld` is more preferable.